tests/test_utils.py

```diff
@@ -0,0 +1,27 @@
+import unittest
```

Comment:
Sample test files using unittest was a mistake from my side %)
No need for lots of unitest boilerplate code. Let's use ``pytest`` and simple assertions for tests.